lib/zendesk_apps_tools/command.rb

```diff
@@ -66,6 +66,10 @@ def new
 
     desc 'validate', 'Validate your app'
     shared_options(except: [:unattended])
+    method_option :'no-deprecate', type: :boolean,
```

Comment:
`--override-deprecate` or `--skip-deprecate`?

Could also have special-cased `--unattended` for this, since it was added to support deployment

Reply:
good idea.